tests/simple/simple.go

```diff
@@ -216,24 +217,33 @@ func (r *runConfig) RunTest(t *spb.SimpleTest) error {
 		// TODO: validate that the inferred type is compatible
 		// with the expected value, if any, in the eval matcher.
 	}
-	// TODO: if check phase disabled, check anyhow and ensure it fails.
 
 	// Eval
-	var ereq exprpb.EvalRequest
-	if checkedExpr == nil {
-		ereq = exprpb.EvalRequest{
+	if !r.checkedOnly {
+		err = r.RunEval(t, &exprpb.EvalRequest{
 			ExprKind:  &exprpb.EvalRequest_ParsedExpr{ParsedExpr: parsedExpr},
 			Bindings:  t.Bindings,
 			Container: t.Container,
+		})
+		if err != nil {
+			return err
 		}
-	} else {
-		ereq = exprpb.EvalRequest{
-			ExprKind:  &exprpb.EvalRequest_CheckedExpr{CheckedExpr: checkedExpr},
+	}
+	if checkedExpr != nil {
+		err = r.RunEval(t, &exprpb.EvalRequest{
+			ExprKind:  &exprpb.EvalRequest_ParsedExpr{ParsedExpr: parsedExpr},
```

Comment:
Did you mean to send a `checkedExpr` here instead?

Reply:
I did indeed mean to do that. Fixed.